src/components/accordion-disclosure/accordion-disclosure.module.css

```
@@ -1,97 +1,107 @@
-    @media (prefers-reduced-motion: no-preference) {
-      transition: transform 0.2s;
-    }
-  }
+		/* Only enable animation if query is supported and value is no-preference */
+		@media (prefers-reduced-motion: no-preference) {
+			transition: transform 0.2s;
+		}
+	}
 
-  /* aka AccordionDisclosure is open */
-  &[aria-expanded='true'] {
-    box-shadow: none;
+	/* aka AccordionDisclosure is open */
+	&[aria-expanded='true'] {
+		box-shadow: none;
 
-    & svg {
-      transform: rotate(90deg);
-    }
-  }
+		& svg {
+			transform: rotate(90deg);
+		}
+	}
 
-  /* aka AccordionDisclosure is closed */
-  &[aria-expanded='false'] {
-    &:hover {
-      box-shadow: var(--token-surface-high-box-shadow);
-      cursor: pointer;
-    }
-  }
+	/* aka AccordionDisclosure is closed */
+	&[aria-expanded='false'] {
+		&:hover {
+			box-shadow: var(--token-surface-high-box-shadow);
+			cursor: pointer;
+		}
+	}
 
-  /*
-  TODO: not composing focus styles here because they use a border. Long term
-  goal is to refactor those to use a box-shadow only like in this situation.
-  */
-  &:focus,
-  &:focus-visible {
-    outline: transparent;
-  }
+	/*
+	TODO: not composing focus styles here because they use a border. Long term
+	goal is to refactor those to use a box-shadow only like in this situation.
+	*/
+	&:focus,
+	&:focus-visible {
+		outline: transparent;
+	}
 
-  &:focus-visible::before {
-    border-radius: 6px;
-    bottom: 0;
-    box-shadow: var(--token-focus-ring-action-box-shadow);
-    content: '';
-    left: 0;
-    position: absolute;
-    right: 0;
-    top: 0;
-    z-index: -1;
-  }
+	&:focus-visible::before {
+		border-radius: 6px;
+		bottom: 0;
+		box-shadow: var(--token-focus-ring-action-box-shadow);
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		z-index: -1;
+	}
 
-  /* Only enable animation if query is supported and value is no-preference */
-  @media (prefers-reduced-motion: no-preference) {
-    transition: box-shadow 0.2s;
-  }
+	/* Only enable animation if query is supported and value is no-preference */
+	@media (prefers-reduced-motion: no-preference) {
+		transition: box-shadow 0.2s;
+	}
 }
 
 .labelContainer {
-  margin-right: 16px;
-  width: 100%;
+	margin-right: 16px;
+	width: 100%;
 }
 
 .title {
-  color: var(--token-color-foreground-strong);
-  display: block;
+	color: var(--token-color-foreground-strong);
+	display: block;
 }
 
 .description {
-  color: var(--token-color-foreground-faint);
-  display: block;
-  margin-top: 4px;
+	color: var(--token-color-foreground-faint);
+	display: block;
+	margin-top: 4px;
 }
 
 .content {
-  padding: 16px;
-  padding-top: 0;
+	padding: 16px;
+	padding-top: 0;
+}
+
+.nested {
+	box-shadow: var(--token-surface-base-box-shadow) !important;
+	& .root-expanded {
+		box-shadow: var(--token-surface-base-box-shadow) !important;
+	}
+	& .button {
+		box-shadow: none !important;
+	}
 }
```

Testing locally, I think this class works without the `!important`s. Would also appreciate some empty lines between the blocks. 🙏🏻

```suggestion
.nested {
	box-shadow: var(--token-surface-base-box-shadow);

	& .root-expanded {
		box-shadow: var(--token-surface-base-box-shadow);
	}

	& .button {
		box-shadow: none;
	}
}
```

---

Oops, I moved this entire block down specifically to not-need the `!important`, but forgot to remove them 😅